src/Sylius/Bundle/ShippingBundle/Behat/ShippingContext.php

```diff
@@ -94,4 +95,12 @@ public function theShippingMethodTranslationsExist(TableNode $table)
 
         $manager->flush();
     }
+
+    /**
+     * @Then The code field should be disable
+     */
+    public function theCodeFileShouldBeDisable()
+    {
+        $this->assertSession()->elementAttributeContains('css', '#sylius_shipping_category_code', 'disabled', 'disabled');
```

Comment:
Can we think about searching field by sth more generic, to detect every "code" field, not only this for ``sylius_shipping_category``?